cfme/services/catalogs/orchestration_template.py

```diff
@@ -140,8 +140,10 @@ def create(self, content):
         view = navigate_to(self, "AddTemplate")
         if(self.template_type == "CloudFormation Templates"):
             temp_type = "Amazon CloudFormation"
-        else:
+        elif(self.template_type == "Heat Templates"):
```

Comment:
there no longer is a default case, should there be a catch all or an error?

if there is a chance this mapping will grow, we should consider putting the type mapping into an actual dictionary